packages/projection/test/operators/Mappers/withHandles.test.ts

```diff
@@ -288,4 +290,40 @@ describe('withHandles', () => {
       expect(handles[0].latestOwnerAddress).toBe(maryAddress);
     });
   });
+
+  describe('subhandles', () => {
+    const project = (tx: Cardano.OnChainTx) =>
+      firstValueFrom(
+        of({
+          block: {
+            body: [tx],
+            header: mockProviders.ledgerTip
+          }
+        } as ProjectionEvent).pipe(withUtxo(), withCIP67(), withHandles({ policyIds: [handlePolicyId] }, logger))
+      );
+
+    it('adds parentHandle data for virtual subhandles that resolves to the parent address', async () => {
+      const { handles } = await project({
+        body: { outputs: [virtualSubHandleOutput] },
+        inputSource: Cardano.InputSource.inputs
+      } as Cardano.OnChainTx);
+
+      expect(handles).toHaveLength(1);
+
+      expect(handles[0].parentHandle).toBe(
+        'addr_test1qpadxfxylvy8p8wejlmt9wnesr2squgr524r77n7hz6yh3h34r3hjynmsy2cxpc04a6dkqxcsr29qfl7v9cmrd5mm89qqh563f'
```

Comment:
`parentHandle` has type of `Handle`. Why is it an address? :thinking: 
```suggestion
        'handl'
```

Reply:
addressed: 998e07df7ceb955a5f76928d37050d083073d390